hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/miscs/AbstractActionTypeOptionGroupLayout.java

```diff
@@ -82,6 +90,51 @@ protected void addSoftItemWithLabel() {
         addComponent(softLabel);
     }
 
+    protected void addDownloadOnlyItemWithLabel() {
+        final FlexibleOptionGroupItemComponent downloadOnlyItem = actionTypeOptionGroup
+                .getItemComponent(ActionTypeOption.DOWNLOAD_ONLY);
+        downloadOnlyItem.setId(UIComponentIdProvider.ACTION_DETAILS_DOWNLOAD_ONLY_ID);
+        downloadOnlyItem.setStyleName(STYLE_DIST_WINDOW_ACTIONTYPE);
+        addComponent(downloadOnlyItem);
+        final Label downloadOnlyLabel = new Label();
+        downloadOnlyLabel.setSizeFull();
+        downloadOnlyLabel.setCaption(i18n.getMessage(UIMessageIdProvider.CAPTION_ACTION_DOWNLOAD_ONLY));
+        downloadOnlyLabel.setDescription(i18n.getMessage(UIMessageIdProvider.TOOLTIP_DOWNLOAD_ONLY_ITEM));
+        downloadOnlyLabel.setStyleName("padding-right-style");
+        downloadOnlyLabel.setIcon(FontAwesome.DOWNLOAD);
+        addComponent(downloadOnlyLabel);
+    }
+
+    protected void addAutoForceItemWithLabelAndDateField(final DateField forcedTimeDateField) {
```

Comment:
I would not define this method here, because it really belongs to `ActionTypeOptionGroupAssignmentLayout` class, and OptionGroup is built together with the DateField, so you don't need to use the method parameter.